charts/umbrella/values.yaml

```diff
@@ -207,6 +207,93 @@ portal:
       senderEmail: "smtp@tx.test"
       password: ""
     portalmigrations:
+      seeding:
+        useDefault: true
+        testDataEnvironments: ""
```

Comment:
I think those values should be removed because that's the default in the portal chart anyway

```suggestion
```

Reply:
`useDefault` is necessary, as it is not available in portal 2.2.0 [here](https://github.com/eclipse-tractusx/portal/blob/portal-2.2.0/charts/portal/values.yaml#L615).

Reply:
@CDiezRodriguez yes, right because the value only gets introduced with portal 2.3.0, the value isn't part of 2.2.0 so it also shouldn't be enabled